packages/world/gas-report.txt

```diff
@@ -9,6 +9,7 @@
 (test/KeysWithValueModule.t.sol) | install keys with value module [world.installRootModule(keysWithValueModule, abi.encode(sourceTableId))]: 600319
 (test/KeysWithValueModule.t.sol) | set a field on a table with KeysWithValueModule installed [world.setField(namespace, sourceFile, keyTuple1, 0, abi.encodePacked(value1))]: 177731
 (test/KeysWithValueModule.t.sol) | change a field on a table with KeysWithValueModule installed [world.setField(namespace, sourceFile, keyTuple1, 0, abi.encodePacked(value2))]: 142076
+(test/World.t.sol) | Call an autonomous system [world.call("namespace", "testSystem", abi.encodeWithSelector(WorldTestSystem.emitCallType.selector))]: 8797746687694640000
```

Comment:
It's weird that `gasleft()` has the same effect here (large gas amount).

Is there a minimum repro we can report to the foundry team?